sdk/src/manifest_store.rs

```diff
@@ -155,6 +155,39 @@ impl ManifestStore {
             .await
             .map(|store| Self::from_store(&store, &mut validation_log))
     }
+
+    /// Asynchronously loads a manifest from a buffer holding a binary manifest (.c2pa) and validates against an asset buffer
+    ///
+    /// # Example: Creating a manifest store from a .c2pa manifest and validating it against an asset
+    /// ```
+    /// use c2pa::{Result, ManifestStore};
+    ///
+    /// fn main() -> Result<()> {
```

Comment:
I'd encourage you to use `#` markers to hide the outer function wrapper and possibly the async block.

See https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html#hiding-portions-of-the-example.

Not a blocker for PR IMHO.

Reply:
Cool - will update.